go/websocketbridge/main.go

```diff
@@ -0,0 +1,133 @@
+package websocketbridge
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/structs"
+	uuid "github.com/satori/go.uuid"
+
+	"golang.org/x/net/websocket"
+)
+
+// Config contains the configuration values for ExoRelay
+type Config struct {
+	Host string
+	Port int
+	Role string
+}
```

Comment:
We can just use the exorelay.Config instead of creating a config with the same structure